### PR TITLE
Shank 382 docs

### PR DESCRIPTION
**Pull request by bschofield-va**

### CODE REVIEW

**Comment by bschofield-va** [comment]
@dbrown-va I've updated the documentation, could you give it a read as someone with no background to see if you have outstanding questions?

File: bin/pingdom [hunk 1]
```diff
@@ -36,12 +36,12 @@ save-check --template <template> --template-args key=value ...
   Process a template and save it, creating a new one or updating an existing entry by the same check name.
 
 Example:
-$0 save-check \
-  --template fhir-resource \
-  -a name=production-dstu2-medication-statement \
-  -a host=api.va.gov \
-  -a url="/services/fhir/v0/dstu2/MedicationStatement?patient=1011537977V693883" \
-  -a authorization_token="\$ARGONAUT_TOKEN" \
+$0 save-check \\
```
Comment (jhulbert-va) [inline bin/pingdom]: Should we have other examples for checks that arent `save-check`.

File: README.md [hunk 2]
```diff
@@ -72,27 +72,148 @@ Pingdom will be integrated with
 - `build.sh` defines the checks and is executed as part of the automated build.
   This script also defines integration IDs.
 - `bin/pingdom` provides basic ability to interact with the Pingdom API.
+- `bin/get-secret` is a utility to extract secrets from AWS Parameter store with the ability
+   to try local secrets first.
 - `templates/*.json` defines check templates in Pingdom API format.
   These checks are processed to substitute template variables and then uploaded
   to Pingdom to create or update checks.
+- `pingdom-checks/*.ping` collection of checks, typically one related group of checks per file.
+  These checks will be processed and updated Pingdom.
```
Comment (jhulbert-va) [inline README.md]: "updated in Pingdom."

File: README.md [hunk 3]
```diff
@@ -72,27 +72,148 @@ Pingdom will be integrated with
...
 ### Create a new check or update an existing
-Update the `build.sh` to include a new check definition or change an existing check.
-Updates to checks are performed by _name_. Name changes are seen as new checks.
+Create or update `pingdom-checks/*.ping` file to include a new check definition or change an 
+existing check.
+
+> Pro Tip: Create a curl command of the check _first_. Once you done this, translating it
+> to Pingdom is fairly straight forward.
+
+Pingdom checks use a _template_ to create the HTTP request. Many checks follow the same pattern,
```
Comment (jhulbert-va) [inline README.md]: Probably should mention the templates are in the `/templates` directory of the repo.

File: README.md [hunk 4]
```diff
@@ -72,27 +72,148 @@ Pingdom will be integrated with
...
+> you may use `$NO_INTEGRATIONS` or `$NO_USERS` to satisfy the template. 
+
+
 
 
 ### Running locally
-if `secrets.conf` file is present, it will be sourced.
+
+> Note: Since there is only one Pingdom instance, testings changes locally is difficult.
+A trial account can be used.
```
Comment (jhulbert-va) [inline README.md]: ~~Maybe a blurb about our _hack_ for finding the integration id of a new trial account?~~

Link to the blurb below?

File: README.md [hunk 5]
```diff
@@ -72,27 +72,148 @@ Pingdom will be integrated with
...
+have permission to access the parameter store.
+
+You can test your local secrets with the `check-local-secrets` utility.
+
+
+
 
 
 # What's done by hand?
 Unfortunately, our robots can't do 100% of activities. At least not yet.
```
Comment (jhulbert-va) [inline README.md]: **YET** dun dun DUN!

File: bin/pingdom (hunk 1 above)
Comment (bschofield-va) [reply bin/pingdom]: nah ...

File: README.md (hunk 4 above)
Comment (bschofield-va) [reply README.md]: /shrug

**Pull request merged**